typescript/README.md

```diff
@@ -0,0 +1,140 @@
+# tBTC v2 SDK
+
+[![build](https://img.shields.io/github/actions/workflow/status/keep-network/tbtc-v2/typescript.yml?branch=main&event=push&label=build)](https://github.com/keep-network/tbtc-v2/actions/workflows/typescript.yml)
+[![npm](https://img.shields.io/npm/v/%40keep-network%2Ftbtc-v2.ts)](https://www.npmjs.com/package/@keep-network/tbtc-v2.ts)
+
+tBTC SDK is a TypeScript library that provides effortless access to the
+fundamental features of the tBTC Bitcoin bridge. The SDK allows developers
+to integrate tBTC into their own applications and offer the power of
+trustless tokenized Bitcoin to their users.
+
+**Table of contents:**
+
+- [Quickstart](#quickstart)
+  - [Installation](#installation)
+  - [Usage](#usage)
+- [Contributing](#contributing)
+  - [Prerequisites](#prerequisites)
+  - [Install dependencies](#install-dependencies)
+  - [Build](#build)
+  - [Test](#test)
+  - [Format](#format)
+- [Documentation](#documentation)
+
+## Quickstart
+
+Here you can find instructions explaining how to use the SDK in your own
+project.
+
+### Installation
+
+To install the tBTC SDK in your project using `yarn`, run:
+
+```bash
+yarn add @keep-network/tbtc-v2.ts
+```
+
+If you prefer to use `npm`, do:
+
+```bash
+npm i @keep-network/tbtc-v2.ts
+```
+
+Please note that you will also need to install the
+[ethers v5](https://docs.ethers.org/v5) library to initialize
+a signer or provider. To do so using `yarn`, invoke:
+
+```bash
+yarn add ethers@legacy-v5
+```
+
+To do the same using `npm`, run:
+
+```bash
+npm i ethers@legacy-v5
+```
+
+> The SDK depends on ethers v5. Proper support for newer ethers versions
+> is not guaranteed right now.
```

Comment:
This line is display like a comment. Is this intended?
![Screenshot 2023-11-02 at 16 01 44](https://github.com/keep-network/tbtc-v2/assets/79790586/a90f5266-2bdc-4e64-a2cf-9089110fb8aa)

Reply:
Yep!